src/main/scala/edu/ie3/simona/service/weather/WeatherSource.scala

```diff
@@ -111,7 +118,7 @@ trait WeatherSource {
 
     /* Go and get the nearest coordinates, that are known to the weather source */
     val nearestCoords = idCoordinateSource
-      .getNearestCoordinates(queryPoint, amountOfInterpolationCoords)
+      .getClosestCoordinates(queryPoint, amountOfInterpolationCoords, distance)
```

Comment:
Please add more information to the logging statement in case we don't find enough coordinates. Specifically mention the distance (and its unit) that has been used to find coordinates and that either there are no coordinates or it is out of bounds of the region for which we have weather data.